### ISSUE TITLE
Consider splitting ValueConverters etc. into a "core" package

### ISSUE THREAD

**Issue by robertjf** [DigitalMomentum/MetaMomentum#6]
This would enable projects that use ModelsBuilder in a separate project to the website to access the value converters without having to install the Plugin UI code...

**Comment by dinc5150** [comment]
Hey @robertjf , sorry forgot to get back to you on this. 

Yeah, totally agree. Bugs me too. 

I hadn't done it yet as I would have never got around to releasing it. I always seem to fumble around with nuget package deployment.

At the moment, I'm just "Exclude files from project" in VS, but still annoys me. 

I'll try and get it sorted for the next release.

**Comment by robertjf** [comment]
Hey @dinc5150 - I've got it done for the Open or Closed project if you want to take a look at that... look in the dev branch...

**Comment by dinc5150** [comment]
Thanks @robertjf , I've also got it in Videolizer, but I wasn't happy with how I had done it, so I'll take a look at yours :)

**Comment by dinc5150** [comment]
@robertjf , I'm just looking up the Twitter Card Ratios, and I've found some site saying 1.91:1 but the [Twitter Docs](https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary-card-with-large-image) say 2:1

Do you agree that the image should be resized to 1200x600 for the twitter card?

**Comment by dinc5150** [comment]
@robertjf I spent the night trying to sort out some weird build issues (https://github.com/dotnet/core/issues/6500) but it's now on Myget for testing if your eager to start using it.

**Comment by robertjf** [comment]
> @robertjf , I'm just looking up the Twitter Card Ratios, and I've found some site saying 1.91:1 but the [Twitter Docs](https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary-card-with-large-image) say 2:1
> 
> 
> 
> Do you agree that the image should be resized to 1200x600 for the twitter card?

I was also looking at what SEO Checker does, but if the official docs say 2:1 use that :)

**Comment by robertjf** [comment]
> @robertjf I spent the night trying to sort out some weird build issues (https://github.com/dotnet/core/issues/6500) but it's now on Myget for testing if your eager to start using it. 

Did you find the work-around for this?  There should have been a step in the [build|release].yml files for this but perhaps I forgot to put it in.  Will check.

**Comment by dinc5150** [comment]
> > @robertjf I spent the night trying to sort out some weird build issues ([dotnet/core#6500](https://github.com/dotnet/core/issues/6500)) but it's now on Myget for testing if your eager to start using it.
> 
> Did you find the work-around for this? There should have been a step in the [build|release].yml files for this but perhaps I forgot to put it in. Will check.

Yeah, I found a workaround by adding the following to the .csproj files, but let me know if you know of something that I can do in the build process, as it would be better. 

```
 <Target Name="RemoveLoggingAnalyzer" BeforeTargets="CoreCompile">
    <ItemGroup>
      <Analyzer Remove="@(Analyzer)" Condition="%(FileName) == 'Lucene.Net.CodeAnalysis.CSharp'" />
    </ItemGroup>
  </Target>
```

**Comment by robertjf** [comment]
@dinc5150 add the following directly after the Nuget Restore command (dotnet restore):
```
    - name: Copy lucene analyser
      run: |
        $src = "C:\Users\runneradmin\.nuget\packages\lucene.net\4.8.0-beta00014\analyzers\dotnet\cs"
        $dest = "C:\Users\runneradmin\.nuget\packages\lucene.net\3.0.3\analyzers\dotnet\cs"
        if (!(Test-Path -Path $dest)) {
            New-Item -ItemType directory -Path $dest
        }
        Copy-Item -Path $src\*.dll -Destination $dest
```

**Comment by dinc5150** [comment]
@robertjf I've published the updates onto Nuget (2.1.0).

Note: I've changed the namespace back to MetaMomentum, so as to not break existing installs / models builder classes.
